pkg/server/delta/v3/server.go

```diff
@@ -184,16 +192,9 @@ func (s *server) processDelta(str stream.DeltaStream, reqCh <-chan *discovery.De
 			s.subscribe(req.GetResourceNamesSubscribe(), &watch.state)
 			s.unsubscribe(req.GetResourceNamesUnsubscribe(), &watch.state)
 
-			watch.responses = make(chan cache.DeltaResponse, 1)
+			watch.responses = watches.deltaMuxedResponses
```

Comment:
I don't think `responses` is actually needed anymore in this case, as it was mostly used to keep track of closing to clean-up the other goroutine

Reply:
Yes, we should just remove it.